### ISSUE TITLE
[Bug]: Unable to set aria-label for sp-progress-bar and sp-progress-circle

### ISSUE THREAD

**Issue by lehelen19** [adobe/spectrum-web-components#4151]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-progress-bar, sp-progress-circle

### Expected behavior

When setting the `aria-label` for a component, it should show up in the DOM and be able to be read by screen readers.

### Actual behavior

We are unable to set an `aria-label` for these components unless a `label` is explicitly set.

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

1. Go to the code playground for sp-progress-circle or sp-progress-bar: https://studio.webcomponents.dev/edit/LfliuY0UocICDCBr21uy?p=stories https://studio.webcomponents.dev/edit/WjcwSATAB2ZGd0FopIyP?p=stories
2. Fork and add an `aria-label` attribute to an instance of the component.
3. Inspect the element and observe the lack of an `aria-label`.

### Sample code that illustrates the problem

Code snippet:
`<sp-progress-circle indeterminate aria-label="progress-circle"></sp-progress-circle>`
DOM result:
`<sp-progress-circle indeterminate="" dir="ltr" size="m" role="progressbar"></sp-progress-circle>`

Code snippet:
`<sp-progress-circle indeterminate label="progress-circle"></sp-progress-circle>` 
DOM result:
`<sp-progress-circle indeterminate="" label="progress-circle" dir="ltr" size="m" role="progressbar" aria-label="progress-circle"></sp-progress-circle>`

**Comment by Westbrook** [comment]
Possible approach from out of band conversation:
> I think we can add logic to the `else` that only removes `aria-label` if it's value and the value of `changes.get('label')` are the same. But, happy to have better suggestions on this as you dig deeper.
